### PR TITLE
add a section for js library

### PR DESCRIPTION
**Pull request by maximousblk**

### CODE REVIEW

**Comment by maximousblk** [comment]
#### 📝 Changed routes:


| [**/**](https://three-em-website-g15tsxfyf-th8ta.vercel.app/) |
|:-:|
|<a href="https://three-em-website-g15tsxfyf-th8ta.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/Dq_WbV_UKQI-wd92_Nsd5ctqloOz4bkYIJzVViTr6yzF4oCCEdFMOPVDPCSFrwXmATYINTRA9vvhTtBze0KjBw.png">(view full size)</a> |







Commit 889c114f3f300811e6be53820b839db54733773e (https://three-em-website-g15tsxfyf-th8ta.vercel.app).

**Comment by martonlederer** [comment]
Loving that alert at the top!

**Comment by martonlederer** [comment]
TODO: add npm link for the "get started" link in the js library subsection

**Comment by andreespirela** [comment]
LGTM

**Review by andreespirela — APPROVED** [review]

File: src/components/Nav.tsx [hunk 1]
```diff
@@ -19,8 +19,14 @@ export default function Nav() {
...
   return (
     <>
-      <div className={styles.Gradient1} />
-      <div className={styles.Gradient2} />
+      <div className={styles.NavGradientWrapper}>
+        <div className={styles.Gradient1} />
+        <div className={styles.Gradient2} />
+      </div>
+      <a className={styles.Banner} href="#library">
+        <span>🎉 We have just launched our Javascript library</span>
```
Comment (t8) [inline src/components/Nav.tsx:27]:
```suggestion
        <span>🎉 We just launched our Javascript library</span>
```

File: src/pages/index.tsx [hunk 2]
```diff
@@ -268,6 +271,49 @@ export default function Home() {
...
+          <h2>A new Frontier</h2>
+          <p>
+            Made for Developers, 3EM can be used in both client and server side.
+            With our JS library, 3EM can be integrated in a variety of
+            applications while still keeping the integrity of our core
+            principles: Blazingly fast, multi-language, secured & deterministic
+            smart contract executor.
+          </p>
+          <a
+            href="https://github.com/three-em/3em"
```
Comment (t8) [inline src/pages/index.tsx:291]: Can we update this link to: https://github.com/three-em/3em/tree/main/js#executecontract

**Review by t8 — CHANGES_REQUESTED** [review]

File: src/pages/index.tsx (hunk 2 above)
Comment (maximousblk) [reply src/pages/index.tsx]: done 👍

**Review by t8 — APPROVED** [review]
LGTM! Amazing work @maximousblk :)

**Pull request merged**